ch1-introduction/custom.css

```diff
@@ -0,0 +1,12 @@
+details summary > * {
+  display: inline;
+}
+
+p {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+}
+
+details {
+    margin-bottom: 2em;
+}
```

Comment:
Think this needs a blank line at the end of the file, otherwise some browsers may not read line 12 in (i.e. line 12 needs a carriage return).